ts/features/barcode/components/BarcodeScanBaseScreenComponent.tsx

```diff
@@ -309,14 +263,56 @@ const BarcodeScanBaseScreenComponent = ({
   const shouldDisplayTorchButton =
     cameraPermissionStatus === "granted" && hasTorch;
 
-  const torchIconButton: ComponentProps<typeof BaseHeader>["customRightIcon"] =
-    {
-      iconName: isTorchOn ? "lightFilled" : "light",
-      accessibilityLabel: isTorchOn
-        ? I18n.t("accessibility.buttons.torch.turnOff")
-        : I18n.t("accessibility.buttons.torch.turnOn"),
-      onPress: handleTorchToggle
-    };
+  const closeButton: HeaderActionProps = {
+    icon: "closeLarge",
+    onPress: navigation.goBack,
+    accessibilityLabel: I18n.t("global.buttons.close")
+  };
+
+  const helpAction: HeaderActionProps = {
+    icon: "help",
+    onPress: startSupportRequest,
+    accessibilityLabel: I18n.t(
+      "global.accessibility.contextualHelp.open.label"
+    ),
+    accessibilityHint: I18n.t("global.accessibility.contextualHelp.open.hint"),
+    testID: "helpButton"
+  };
+
+  const torchAction: HeaderActionProps = {
+    icon: isTorchOn ? "lightFilled" : "light",
+    accessibilityLabel: isTorchOn
+      ? I18n.t("accessibility.buttons.torch.turnOff")
+      : I18n.t("accessibility.buttons.torch.turnOn"),
+    onPress: handleTorchToggle
+  };
+
+  const HeaderTransparent = () => (
+    <View
+      style={{
+        paddingHorizontal: IOVisualCostants.appMarginDefault,
+        height: IOVisualCostants.headerHeight,
+        flexGrow: 1,
+        flexDirection: "row",
+        alignItems: "center",
+        justifyContent: "space-between"
+      }}
+    >
+      <IconButton {...closeButton} color="contrast" />
+
+      <HStack allowScaleSpacing space={16} style={{ flexShrink: 0 }}>
+        {shouldDisplayTorchButton ? (
+          <IconButton {...torchAction} color="contrast" />
+        ) : null}
```

Comment:
What do you think to apply the logical AND operator here?


```suggestion
        {shouldDisplayTorchButton && <IconButton {...torchAction} color="contrast" />}
```